src/components/about/About.module.css

```diff
@@ -0,0 +1,84 @@
+.about {
+  max-width: 1280px;
+  padding: 30px 30px 40px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  row-gap: 30px;
+}
+
+.about__image {
+  width: 100%;
+  height: 294px;
+  object-fit: cover;
+  border-radius: 30px;
+}
+
+.about__content {
+  width: 100%;
+}
+
+.about__title {
+  margin: 0 0 30px;
+  color: var(--white);
+  font-size: 27px;
+  font-weight: 600;
+}
+
+.about__description {
+  margin: 0 0 30px;
+  color: var(--white);
+  font-size: 16px;
+}
+
+.about__link {
+  color: var(--white);
```

Comment:
Hey, @snischuk, thanks for a good pr. 
The animation for the link looks cute, but, could you please add the underline decoration, as on the design?
![image](https://github.com/snischuk/fe-react-2024/assets/98031768/cc49fc3c-d89b-4d29-8dd5-42ed05d35c94)

Reply:
Ok, as this is the only one my comment, I'm going to merge this pr. Please correct in the next one.

Reply:
Ok, i'll fix it, thanks for review!